reV/supply_curve/sc_aggregation.py

```diff
@@ -249,27 +249,31 @@ def __init__(self, excl_fpath, tm_dset, econ_fpath=None,
         excl_fpath : str | list | tuple
             Filepath to exclusions data HDF5 file. The exclusions HDF5
             file should contain the layers specified in `excl_dict`
-            and `data_layers` (though data for the latter may be
-            stored in a separate file - see the `data_layers` input
-            documentation for more details). These data layers may
-            be spread out across multiple files, in which case this
-            input should be a list or tuple of filepaths to multiple
-            exclusion HDF5 files containing the layers. Note that each
-            data layer must be uniquely defined (i.e. only appear once
-            and in a single input file).
+            and `data_layers`. These layers may also be spread out
+            across multiple HDF5 files, in which case this input should
+            be a list or tuple of filepaths pointing to the files
+            containing the layers. Note that each data layer must be
+            uniquely defined (i.e.only appear once and in a single
+            input file).
         tm_dset : str
             Dataset name in the `excl_fpath` file containing the
-            techmap (exclusions-to-resource mapping data). This dataset
-            uniquely couples the (typically high-resolution) exclusion
-            layers to the (typically lower-resolution) resource data,
-            and therefore should be unique for every new resource data
-            set that is paired with the exclusion data. If running
-            ``reV`` from the command line, you can specify a name that
-            is not in the exclusions HDF5 file, and ``reV`` will
-            calculate the techmap for you. Note however that computing
-            the techmap and writing it to the exclusion HDF5 file is a
-            blocking operation, so you may only run a single ``reV``
-            aggregation step at a time this way.
+            techmap (exclusions-to-resource mapping data). This data
+            layer links the supply curve GID's to the generation GID's
+            that are used to evaluate performance metrics such as
+            ``mean_cf``.
+
+            .. Important:: This dataset uniquely couples the (typically
+              high-resolution) exclusion layers to the (typically
+              lower-resolution) resource data. Therefore, a separate
+              techmap must be used for every unique combination of
+              resource/exclusion data.
```

Comment:
More precise to say "every unique combo of resource meta data + exclusion meta data" or something.... you know what i mean? It's the coordinate/meta data mappings that matter.

Reply:
Updated to "every unique combination of resource and exclusion coordinates". Does that work? Or did you want me to mentione `meta` in particular?

Reply:
Perfect